src/SFA.DAS.ProviderApprenticeshipsService.Web/Views/ManageApprentices/EditStartedApprenticeship.cshtml

```diff
@@ -129,7 +129,7 @@
             }
 
             @*todo: better for view to distill down to single flag similar to IsEndDateLockedForUpdate*@
-            @if (Model.IsLockedForUpdate || Model.IsUpdateLockedForStartDateAndCourse || Model.IsContinuation)
+            @if (Model.IsLockedForUpdate || Model.IsUpdateLockedForStartDateAndCourse)
```

Comment:
This change reverses a change in the previous branch. Was the previous logic wrong or is the logic now in das-commitments so we don't need to stop users editing here